CHANGES/9566.bugfix

```diff
@@ -0,0 +1,9 @@
+Scenario:
```

Comment:
Hey!  The changelog entries are usually short descriptions, could you replace all of this something more like:

"Fixed a bug where sub-repos (distribution tree repos) could conflict with each other in common workflows."

Reply:
@HolgerHees , great job at finding not very straightforward issues and solution for them! Thanks a lot